### PR TITLE
Build Script

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#6 
Add scripts and some minor edits

### CODE REVIEW

File: autopep8.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+python -m autopep8 nafas --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
+python -m autopep8 setup.py --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
+python -m autopep8 otherfiles --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose
```
Comment (sadrasabouri) [inline autopep8.bat:3]: Could you pleas make `.sh` version of this file too, anyway I think I can add it to your `build_script` branch too.
I don't know if it's necessary or not but there is a newline miss here.

File: build_exe.bat [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+python --version | find /i "Python 3.4" > nul
+echo -----
+if errorlevel 1 (
+	echo Warning : Please use Python 3.4.x
+) else (
+	echo Python version check done!
+)
+echo -----
+python -m PyInstaller NAFAS.spec
+pause
```
Comment (sadrasabouri) [inline build_exe.bat:11]: According to last comment there is another newline miss here too.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thank you I've left you some comments, hope they'll be useful.

File: build_exe.bat (hunk 2 above)
Comment (sepandhaghighi) [reply build_exe.bat]: Solved in https://github.com/sepandhaghighi/nafas/pull/10/commits/1c8c9c154b68c2094e5e9327b11ad7aa263ff325

File: autopep8.bat (hunk 1 above)
Comment (sepandhaghighi) [reply autopep8.bat]: Solved in https://github.com/sepandhaghighi/nafas/pull/10/commits/1c8c9c154b68c2094e5e9327b11ad7aa263ff325

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**